docs/products/clickhouse/howto/manage-dbs-tables.rst

```diff
@@ -0,0 +1,57 @@
+
+.. note::
+
+    There are other installation options available for ClickHouse clients for certain operating systems. You can find them `in the official ClickHouse documentation <https://clickhouse.com/docs/en/integrations/sql-clients/clickhouse-client-local>`_.
+    Our managed service **Aiven for ClickHouse** has some minor differences comparing to "classical" ClickHouse. These difference are needed to ensure security, stability and proper replication of data.
+
+Aiven for ClickHouse® takes care of running queries in a distributed mode affecting entire cluster. In the "classical" ClickHouse the queries ``CREATE``, ``ALTER``, ``RENAME`` and ``DROP`` will only affect the server where they are run. In contrast, we ensure the proper distribution across all cluster machines behind the scenes. You don't need to remember using ``ON CLUSTER`` for every query.
+
+Create a database
+-----------------
+
+Creating databases in an Aiven for ClickHouse service can only be done via the Aiven platform; the `admin` user is not allowed to create databases directly for security and reliability reasons. However, you can create a new database through the web interface of `Aiven console <https://console.aiven.io/>`_:
+
+#. Log in to `Aiven console <https://console.aiven.io/>`_ and select your service.
+#. On the service page, open the tab *Databases & Tables*.
+#. Enter your database name in *Create a new database* and select **Create database**.
+
+   You'll see the name of the database appear in the *Database List* section.
+   On our side, we enable necessary customizations and run secondary queries to grant access to the admin user.
+
+Remove a database
+-----------------
+
+Similar to creating the database, removal should also be done through the Aiven platform. In the web interface of `Aiven console <https://console.aiven.io/>`_ you'll find delete button next to the database you created in the *Database List*.
+
+.. note::
+
+    If you try adding or removing a database in **Aiven for ClickHouse** through the command line, you'll encounter an exception ``Not enough privileges.(ACCESS_DENIED)``. Please use the Aiven web interface to add or remove a database.
+
+Create a table
+--------------
+
+Tables can be added with an SQL query, either with the help of :doc:`the web query editor <../howto/use-query-editor>` or with :doc:`the cli <../howto/use-cli>`. In both cases the SQL query will look the same. For example, below is a query to add a new table ``expenses`` to ``transactions`` database. To keep it simple this example has unrealistically small amount of columns:
+
+.. code:: sql
+
+        CREATE TABLE transactions.expenses (
+            Title String,
+            Date DateTime,
+            UserID UInt64,
+            Amount UInt32
+        )
+        ENGINE = ReplicatedMergeTree ORDER BY Date;
```

Comment:
I don't understand, should we encourage the user to create `ReplicatedMergeTree` tables (as this implies), or encourage them to simply use the `MergeTree` engine (since it will get re-written anyway)?

Reply:
@aris-tritas-aiven, This is how it's been defined so far. I'm keeping it as is. Please let me know if you think we need to change that